demo/src/screens/incubatorScreens/IncubatorTextFieldScreen.tsx

```diff
@@ -191,7 +191,7 @@ export default class TextFieldScreen extends Component {
 
           <TextField
             label="Email"
-            labelColor={{default: Colors.grey10, focus: Colors.blue20, error: Colors.red30, disabled: Colors.grey40}}
+            labelColor={{default: Colors.$textDefault, focus: Colors.$textGeneral, disabled: Colors.$textNeutralLight}}
```

Comment:
Missing `error`?

Reply:
red30 is the default so we don't really need to pass it here

Reply:
No, it's not, you can test it in the screen

Reply:
Right, I thought it was the color of the validation error but it's the label's one.. 
Fixed